clusters/app.ci/cert-manager/registry-ci-openshift-org_certificate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  labels:
+    gcp-project: openshift-ci-infra
+  name: registry-ci-openshift-org
+  namespace: openshift-image-registry
+spec:
+  secretName: customized-route-tls
```

Comment:
Lets please just name it like the cert, with a `-tls` suffix. `customized-route` doesn't really make anything clear either